lnd.go

```diff
@@ -614,6 +618,14 @@ func Main(cfg *Config, lisCfg ListenerCfg, interceptor signal.Interceptor) error
 			// The channel is buffered by one element so writing
 			// should not block here.
 			walletInitParams.MacResponseChan <- adminMacBytes
+
+			if lisCfg.RPCListeners != nil {
```

Comment:
The `range` command is nil-safe, so we don't need the extra `if`. And I think with the check if the mac channel is nil, we can just loop over `grpcListeners` directly.